examples/fourier_transform.py

```diff
@@ -28,7 +28,11 @@
     x /= np.linalg.norm(x)
     x_hat = G.gft(x).squeeze()
 
-    assert np.all((-limit < x) & (x < limit))
+    try:
+        assert np.all((-limit < x) & (x < limit))
+    except AssertionError as msg: 
```

Comment:
Does it make sense to catch the `AssertionError`? And perhaps more importantly, why is there an `assert` in the original code?

Reply:
An assert is used in that part of the code to check if a series of randomly generated values `[x] is within a certain limit band `[-limit, limit], it will only plot values inside that band, the code has been modified to include the plot part in the fix.